src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -443,6 +443,13 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Lucene.IndexInputType", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public LuceneIndexInputType LuceneIndexInput { get; set; }
         
+        [Description("Max time to wait when forcing the storage environment flush and sync when replacing side-by-side index.")]
+        [DefaultValue(30)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.MaxTimeToWaitAfterFlushAndSyncWhenReplacingSideBySideIndexInSec", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
+        public TimeSetting MaxTimeToWaitAfterFlushAndSyncWhenReplacingSideBySideIndexInSec { get; protected set; }
```

Comment:
property is a TimeSetting so it is a TimeSpan, hence there is no need to put the 'InSec' there
ConfigurationEntry is set by string, so from user experience point of view, it is better to include the unit there